data/p3/prepare_p3.py

```diff
@@ -0,0 +1,439 @@
+from functools import partial
+import os
+import multiprocessing
+from datasets import load_dataset, load_from_disk
+import jsonlines
+
+os.environ["HF_DATASETS_CACHE"] = "/gpfsscratch/rech/six/commun/datasets"
+
+"""Get task list:
+!git clone https://github.com/bigscience-workshop/t-zero.git
+%cd t-zero
+!pip install -e .[seqio_tasks]
+!pip install -q py7zr
+
+import t0.seqio_tasks
+import seqio
+tasks = [task.name for task in seqio.MixtureRegistry.get('t0_train').tasks]
+print(tasks)
+
+
+
+After running the script, merge train & validation jsonls separately into two big files:
+cat folder_with_all_jsonl/*.jsonl > merged_file.jsonl
+"""
+TZERO_TASK_LIST = [
```

Comment:
Hum I think we should change the logic:
 - List dataset_name and dataset_config_name
 - Find all the prompts dynamically given that set. 
 
 This can be done by running:
 ```
 DATASET_NAMES_AND_CONFIG [
    ('advercarial_qa', 'dbert')
    ... 
 ]
 
from datasets import get_dataset_infos

infos = get_dataset_info("bigscience/P3")

# find all configs that match with f"{elt[0]}_{elt[1]}*"
 ```

Reply:
I can't find any information about which datasets are T0 / T0+ / T0++ in the `bigscience/P3` dataset. So I think the outlined one is the best way